rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/asset_search/action.py

```diff
@@ -0,0 +1,68 @@
+import ast
+
+import insightconnect_plugin_runtime
+from .schema import AssetSearchInput, AssetSearchOutput, Input, Output, Component
+
+
+# Custom imports below
+
+
+def format_response(resources: [], params={}):
+    results = list()
+    ip = params.get(Input.IP)
+    hostname = params.get(Input.HOSTNAME)
+    for page in resources:
```

Comment:
Agreed with Tyler on this one. Not sure I understand parsing the response if we can use the json response and then reference by keys. Definitely want this to be addressed.